tests/validation/validate_kinetic.hpp

```diff
@@ -23,11 +28,11 @@ void run_kinetic_dt(nest::mc::cell& c, float t_end, nlohmann::json meta, const s
     };
 
     meta["sim"] = "nestmc";
-    meta["backend"] = LoweredCell::backend::name();
+    meta["backend"] = backend==backend_policy::use_multicore? "cpu" : "gpu";
```

Comment:
Agreed, I was thinking that it was a bit fragile when I implemented it. But, as you say, this is placeholder code, and a proper solution would have been overkill.

I will add a `FIXME` to this line.